### PR TITLE
Issue #323: Rename "Admin menu" to "Admin bar"

### PR DESCRIPTION
**Pull request by tarekdj**
PR for https://github.com/backdrop/backdrop-issues/issues/323

### CODE REVIEW

**Comment by tarekdj** [comment]
@quicksketch this PR is based on "git mv" no file deleted or created!

File: core/modules/admin_bar/admin_bar.inc [hunk 1]
```diff
@@ -851,13 +851,17 @@ function _admin_menu_flush_cache($name = NULL) {
...
   // Image source might have been passed in as theme variable.
   if (!isset($variables['src'])) {
-    $favicon = backdrop_get_favicon();
-    $variables['src'] = $favicon['path'];
+    if (theme_get_setting('toggle_favicon')) {
+      $variables['src'] = theme_get_setting('favicon');
+    }
+    else {
+      $variables['src'] = base_path() . 'core/misc/favicon.ico';
+    }
```
Comment (quicksketch) [inline core/modules/admin_bar/admin_bar.inc]: These lines changed as part of #464, as theme_get_setting() no longer is responsible for the location of the favicon. I think this is the only recent change to admin_menu() that will need to be merged up-to-date.

File: core/modules/admin_bar/admin_bar.info [hunk 2]
```diff
@@ -1,6 +1,6 @@
-name = Administration menu
+name = Administration bar
 description = "Provides a dropdown menu to most administrative tasks and other common destinations."
 package = Core
 backdrop = 1.x
-configure = admin/config/administration/admin-menu
-version = BACKDROP_VERSION
+configure = admin/config/administration/admin-bar
+version = BACKDROP_VERSION
```
Comment (quicksketch) [inline core/modules/admin_bar/admin_bar.info]: Minor, but let's keep the new line at the end of the file to be consistent with coding standards.

**Comment by quicksketch** [comment]
This all looks great! Don't forget that we have an update in system.install that enables admin_menu.module (`system_update_1009`).

**Comment by tarekdj** [comment]
@quicksketch if the follow-up is good I'll squash commits and update PR

**Comment by quicksketch** [comment]
Looks good to me! For users updating existing Backdrop installs, it might be a good idea to add a new system_update_x() that just enables admin_bar if it's not enabled already. Although HEAD2HEAD updates can be difficult, we'd like to minimize the need to reinstall whenever possible, and just making sure the module is available would be trivial. Let's not worry about migrating the settings, since users will certainly reinstall before launching a live site, but just something like this would help our existing developers next time they git pull:

```
/**
 * Enable admin_bar, replacing the renamed admin_menu.
 */
system_update_1023() {
  if (!module_exists('admin_bar')) {
    module_enable(array('admin_bar'));
  }
}
```

**Comment by jenlampton** [comment]
I gave this a good solid test, and everything is working great. Thanks @tarekdj!

**Comment by docwilmot** [comment]
@tarekdj  small error. In admin_bar.inc line 514, system_admin_bar_block_page should be system_admin_menu_block_page. That code block is meant to hide empty categories that are appearing in the config menu. ("user interface" and "workflow")
capture2

**Comment by quicksketch** [comment]
Thanks @docwilmot. I should have caught that. I'll file a fix shortly.

**Comment by quicksketch** [comment]
Fixed this in https://github.com/backdrop/backdrop/pull/487. Thanks @docwilmot!

**Pull request merged**